WebCalc/WebCalc/Models/CalculationData.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace WebCalc.Models;
+
+public class CalculationData
+{
+    public float FirstOperand { get; set; }
+    public float SecondOperand { get; set; }
+    public string Sign { get; set; } = "";
```

Comment:
```suggestion
    [RegularExpression("'+'|-|'*'|/", ErrorMessage = "Invalid sign")]
    public string Sign { get; set; } = "";
```
Всегда должна быть валидация. А что, если нам не с нашего клиента зашлют пакет на сервер? С какой либо некорректной строкой? Сервер упадет.
(-1)